MaterialSkin/Controls/MaterialExpansionPanel.cs

```diff
@@ -0,0 +1,607 @@
+using MaterialSkin.Animations;
+using System;
+using System.Configuration;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Text;
+using System.Windows.Forms;
+
+
+namespace MaterialSkin.Controls
+{
+    public class MaterialExpansionPanel : Panel, IMaterialControl
+    {
+
+        #region "Private members"
+
+        private const int _expansionPanelDefaultPadding = 16;
+        private const int _leftrightPadding = 24;
+        private const int _buttonPadding = 8;
+        private const int _expandcollapsbuttonsize = 24;
+        private const int _textHeaderHeight = 24;
+        private const int _headerHeightCollapse = 48;
```

Comment:
_headerHeightCollapse and _headerHeightExpand should be the same. The way they are now, the header is "dancing around" when changing states, and that's a big no no.

Reply:
It might be only a misunderstanding due unappropriate constant name. Better name should be something like _controlHeightCollapse & _controlHeightExpand.
If not then please clarify a bit.